### ISSUE TITLE
Broken example: Draggable Cropping Frame

### ISSUE THREAD

**Issue by dhil** [links-lang/links#578]
The "Draggable Cropping Frame" example is currently broken because the image [http://groups.inf.ed.ac.uk/links/data/17-Heriot-Row.jpeg](http://groups.inf.ed.ac.uk/links/data/17-Heriot-Row.jpeg) is unavailable. The server produces a 503 error.
```
$ curl -v http://groups.inf.ed.ac.uk/links/data/17-Heriot-Row.jpeg
*   Trying 129.215.202.26...
* TCP_NODELAY set
* Connected to groups.inf.ed.ac.uk (129.215.202.26) port 80 (#0)
> GET /links/data/17-Heriot-Row.jpeg HTTP/1.1
> Host: groups.inf.ed.ac.uk
> User-Agent: curl/7.58.0
> Accept: */*
> 
< HTTP/1.1 503 Service Unavailable
< Date: Tue, 30 Apr 2019 09:11:02 GMT
< Server: Apache/2.4
< X-Frame-Options: SAMEORIGIN
< Content-Length: 299
< Connection: close
< Content-Type: text/html; charset=iso-8859-1
< 
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>503 Service Unavailable</title>
</head><body>
<h1>Service Unavailable</h1>
<p>The server is temporarily unable to service your
request due to maintenance downtime or capacity
problems. Please try again later.</p>
</body></html>
* Closing connection 0
```
I am not sure who controls the "links group", @slindley, @jamescheney?

**Comment by jamescheney** [comment]
It is not just this one URL but also http://groups.inf.ed.ac.uk/links/ returns the same 403.  Other groups.inf.ed.ac.uk pages appear to be fine so it is a problem with the links group, possibly something computing support should handle?  

I have never had any involvement with the setup of groups.inf.ed.ac.uk/links.  I suspect it has bitrotted since the shift to using www.links-lang.org.  Probably we should just put the image file on the examples server (or even make it a relative URL and not have a fixed global URL hardwired into the code).

**Comment by dhil** [comment]
> Probably we should just put the image file on the examples server (or even make it a relative URL and not have a fixed global URL hardwired into the code).

Yes, it would be convenient to keep the resources together in a place where everybody has access. I do not have access to the Links group on the inf-network.

> I have never had any involvement with the setup of groups.inf.ed.ac.uk/links.

OK, I will leave it to @slindley.

**Comment by slindley** [comment]
The image is no longer in the groups.inf.ed.ac.uk/links user space as far as I can tell. However, I did manage to find it in a backup of an old SVN repository. I have now pushed it to master. Alas, there is now a bug in the crop example. It appears to display the entire image rather than the part in the box and dragging doesn't seem to work.

**Comment by SimonJF** [comment]
We agreed in the meeting that I would look at this, and that it is a blocker for 0.9.

**Comment by SimonJF** [comment]
Fixed by #679.
